src/Symfony/Component/Config/Definition/EnumNode.php

```diff
@@ -24,13 +24,8 @@ class EnumNode extends ScalarNode
 
     public function __construct($name, NodeInterface $parent = null, array $values = array())
     {
-        $values = array_unique($values);
-        if (count($values) <= 1) {
```

Comment:
I agree. Done in https://github.com/javiereguiluz/symfony/commit/e75a983c42178caf11a99556bf1063fb5e605db8